### DISCUSSION TITLE
ASN or AS and Country Code or Country Name for Allowed clients and Disallowed clients???

### DISCUSSION THREAD

**Discussion by t0ny54** [AdguardTeam/AdGuardHome#4438] [Ideas]
CIDRs, IP addresses, or ClientIDs is too general. Need spesific like Country Code or Country Name for Allowed clients and Disallowed clients.

**Comment by ainar-g** [comment]
This sounds like feature request #1487.  That would require a relatively fresh GeoIP database present, so we're currently not sure how to go about that.
